.github/workflows/docker.yml

```diff
@@ -20,17 +20,19 @@ jobs:
       matrix:
         python-impl:
         - python
+        - pypy
         python-version:
         - '3.7'
         - '3.8'
         - '3.9'
         - '3.10'
-        include:
+        exclude:
+        # XXX: pypy-3.7 is not compatible with Dockerfile.pypy anymore
         - python-impl: pypy
           python-version: '3.7'
-        # XXX: this will still fail (probably because of the code Cython generates on python-rocksdb)
-        # - python-impl: pypy
-        #   python-version: '3.8'
+        # XXX: pypy-3.10 does not exist yet
```

Comment:
Should we remove it then?

Reply:
Yes, that's what is being done now. The diff got a bit confusing, but I basically added an "exclude" section, and it looks like "include" became "exclude".

Reply:
Oh! I got it now. Thanks for the clarification.